zipline/data/minute_bars.py

```diff
@@ -415,11 +413,49 @@ def write(self, sid, df):
                 volume : float64|int64
             index : DatetimeIndex of market minutes.
         """
+        cols = {
+            'open': df.open.values,
+            'high': df.high.values,
+            'low': df.low.values,
+            'close': df.close.values,
+            'volume': df.volume.values,
+        }
+        dts = df.index.values
```

Comment:
In our internal code, this function would no longer be called, in favor of `write_cols`, so that we can avoid the overhead of `DataFrames` altogether when loading data.

This function is left here for case where convenience is valued over performance.

Open to changing this name to something like `write_dataframe`